README.md

```diff
@@ -30,6 +30,13 @@ CA certificates can be added to the java keystore with the following variables:
       password: changeit
 ```
 
+```yaml
+    keep_oracle_jdk: false
+```
+
+This role uninstall OracleJDK by default. You can change the
```

Comment:
Should not `Debian based distributions only` added ?

Reply:
I agree :)